astropy/cosmology/flrw/tests/test_base.py

```diff
@@ -333,12 +347,14 @@ def test_init_m_nu(self, cosmo_cls, ba):
         """
         # Test that it works when m_nu has units.
         cosmo = cosmo_cls(*ba.args, **ba.kwargs)
-        assert np.all(cosmo.m_nu == ba.arguments["m_nu"])  # (& checks len, unit)
+        assert np.all(
```

Comment:
Tests shouldn't be using `np.all`, they should be using [`np.testing.assert_array_equal()`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_equal.html).

Reply:
✅